src/shared/services/auth/services/keycloak.ts

```diff
@@ -0,0 +1,127 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2020 - 2021 Gemeente Amsterdam
+import keycloakJS, { KeycloakInitOptions, KeycloakInstance } from 'keycloak-js'
+import configuration from 'shared/services/configuration/configuration'
+
+const AUTH_REDIRECT_URI = `${global.location.protocol}//${global.location.host}/manage/incidents`
```

Comment:
Why not use `window.location.origin` here as well?

Reply:
`window` is indeed better in this context, changed it

Reply:
I actually meant using `window.location.origin` instead of `${global.location.protocol}//${global.location.host}`

Reply:
Whoops, I read that wrong, fixed it